extensions/fbc/src/org/sbml/jsbml/ext/fbc/converters/FbcV1ToCobraConverter.java

```diff
@@ -136,8 +159,13 @@ public SBMLDocument convert(SBMLDocument sbmlDocument) throws SBMLException {
         }
         
       }
+      // 5. unset List of flux bounds
       fbcModelPlugin.unsetListOfFluxBounds();
     }
+   
+    // TODO: it is unclear whether this should be done
```

Comment:
Here, you should use the disablePackage methods from the SBMLDocument. Might as well be a good place to test the PackageDisabler class that Onur is testing.